blocks/core/ff_module/ff_module-filter/ff_module-filter.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(function(){ 
+	$("[data-control='filter-dropdown']").click(function(){
+		var filterDropdown = $(this),
+		      filterIcon = filterDropdown.find("[data-icon='filter-dropdown-icon']"),
+		      filterContent = $("[data-content='filter-content']");
```

Comment:
Hi Simone, have you tested this with multiple filters on a page? It looks to me like this would select all other `data-content='filter-content'` elements on the page.